lib/oxidized/nodes.rb

```diff
@@ -58,7 +58,8 @@ def show node
     def fetch node, group
       with_lock do
         i = find_node_index node
-        output = self[i].output.new
+        # Use first output type
+        output = self[i].output[0].new
```

Comment:
ACK. We'd need to leave such change to version 1 to change it all around and not accept string. But I think our average user actually prefers the string, and more competent people can adapt to what ever.